reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt

```diff
@@ -89,29 +96,57 @@ class AntennaAttributionDocumentReporter : Reporter {
 
         // Use the default template unless a custom template is provided via the options.
         var templateId = DEFAULT_TEMPLATE_ID
-        options[TEMPLATE_ID]?.let { id ->
-            options[TEMPLATE_PATH]?.let { path ->
-                val templatePath = File(path)
+        var templateFiles = mutableMapOf<String, File>()
 
+        val providedTemplateId = options[TEMPLATE_ID]
+        val providedTemplatePath = options[TEMPLATE_PATH]
+        if (providedTemplatePath != null) {
+            val templatePath = File(providedTemplatePath)
+
+            if (templatePath.isFile && !providedTemplateId.isNullOrBlank()) {
                 require(templatePath.isFile && templatePath.extension == "jar" && templatePath.length() > 0) {
```

Comment:
We can drop the `templatePath.isFile` from here now as it's already checked above.